rust/cloud-storage/opensearch_client/src/search/unified/test.rs

```diff
@@ -398,6 +398,17 @@ fn test_deserialization() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn expected_sort<'a>() -> Vec<SortType<'a>> {
```

Comment:
This doesn't need to allocate, either return a [SortType; 2] array or a &'static  slice